__tests__/app.test.js

```diff
@@ -84,4 +84,49 @@ describe("App", () => {
         });
     });
   });
+
+  describe("/api/reviews/:review_id/comments", () => {
+    test("GET 200- should respond with an array of comments for the given review id.", () => {
+      return request(app)
+        .get("/api/reviews/2/comments")
+        .then((res) => {
+          expect(res.body.comments.length).toBe(3);
+          res.body.comments.forEach((comment) => {
+            expect(typeof comment.comment_id).toBe("number");
+            expect(typeof comment.votes).toBe("number");
+            expect(typeof comment.created_at).toBe("string");
+            expect(typeof comment.author).toBe("string");
+            expect(typeof comment.body).toBe("string");
+            expect(typeof comment.review_id).toBe("number");
+          });
+        });
+    });
+
+    test("GET 200- array should be ordered by created_at ", () => {
+      return request(app)
+        .get("/api/reviews/2/comments")
+        .expect(200)
+        .then((res) => {
+          const arr = res.body.comments;
+          expect(arr).toBeSortedBy("created_at");
+        });
+    });
+
```

Comment:
I would like another test for an ID that is valid, but has no comments yet (so should return an empty array) - take a look at your data and see what ID has no comments :)